### ISSUE TITLE
android have you declared this activity in your AndroidManifest.xml

### ISSUE THREAD

**Issue by CodeGather** [Sh1d0w/multi_image_picker#122]
**Bug Report**

`multi_image_picker` version: 4.1.2

Is it happening on Android: yes

```
 
E/AndroidRuntime(32029): Caused by: android.content.ActivityNotFoundException: Unable to find explicit activity class {com.ki5k.fluttershop/com.sangcomz.fishbun.ui.album.AlbumActivity}; have you declared this activity in your AndroidManifest.xml?
E/AndroidRuntime(32029): 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1932)
E/AndroidRuntime(32029): 	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1615)
E/AndroidRuntime(32029): 	at android.app.Activity.startActivityForResult(Activity.java:4472)
E/AndroidRuntime(32029): 	at android.app.Activity.startActivityForResult(Activity.java:4430)
E/AndroidRuntime(32029): 	at com.sangcomz.fishbun.FishBunCreator.startAlbum(FishBunCreator.java:262)
E/AndroidRuntime(32029): 	at com.vitanov.multiimagepicker.MultiImagePickerPlugin.presentPicker(MultiImagePickerPlugin.java:646)
E/AndroidRuntime(32029): 	at com.vitanov.multiimagepicker.MultiImagePickerPlugin.onRequestPermissionsResult(MultiImagePickerPlugin.java:106)
E/AndroidRuntime(32029): 	at io.flutter.app.FlutterPluginRegistry.onRequestPermissionsResult(FlutterPluginRegistry.java:199)
E/AndroidRuntime(32029): 	at io.flutter.app.FlutterActivityDelegate.onRequestPermissionsResult(FlutterActivityDelegate.java:129)
E/AndroidRuntime(32029): 	at io.flutter.app.FlutterActivity.onRequestPermissionsResult(FlutterActivity.java:137)
E/AndroidRuntime(32029): 	at android.app.Activity.dispatchRequestPermissionsResult(Activity.java:7388)
E/AndroidRuntime(32029): 	at android.app.Activity.dispatchActivityResult(Activity.java:7239)
E/AndroidRuntime(32029): 	at android.app.ActivityThread.deliverResults(ActivityThread.java:4320)
E/AndroidRuntime(32029): 	... 9 more
Lost connection to device.
```

**Describe the bug**

**Comment by Sh1d0w** [comment]
Please provide example repository where the problem can be reproduced.

**Comment by CodeGather** [comment]
> 请提供可以重现问题的示例存储库。

已自行解决，非常谢谢您的回答

**Comment by CodeGather** [comment]
Android problems have been fixed, but a new problem has arisen that the IOS side is not working properly. I've followed the documentation you gave me.
Https://sh1d0w.github.io/multi_image_picker/#/iostroubleshooting
And complete the Xcode settings, but still very sorry to fail to solve the problem.
I've posted the question to the repository, and I hope I can answer it. Thank you very much.
https://github.com/CodeGather/flutter_shop
